proto/provenance/marker/v1/marker.proto

```diff
@@ -208,3 +208,10 @@ message EventSetNetAssetValue {
   string volume = 3;
   string source = 4;
 }
+
+// EventMarkerParamsUpdated event emitted when marker params are updated.
+message EventMarkerParamsUpdated {
```

Comment:
Spec should be updated to include this Event.

Reply:
Yep, good catch! Done.